__tests__/data/iterable_router_input.json

```diff
@@ -35,48 +35,5 @@
       },
       "Enabled": true
     }
-  },
-  {
```

Comment:
One test case has been removed. We should not remove existing test cases.

Reply:
Also, let's add a test case to depict Iterable batching.

Reply:
Added new test cases, depicting batching